Xamarin.Forms.Xaml.UnitTests/Validation/SetterOnNonBP.xaml.cs

```diff
@@ -29,7 +29,7 @@ public class SetterOnNonBPTests
 			public void ShouldThrow (bool useCompiledXaml)
 			{
 				if (!useCompiledXaml)
-					Assert.Throws(new XamlParseExceptionConstraint(10, 13), () => new SetterOnNonBP(useCompiledXaml));
+					Assert.Throws(new BuildExceptionConstraint(10, 13), () => new SetterOnNonBP(useCompiledXaml));
```

Comment:
Looks like this type didn't get checked in

Reply:
forgot to commit. fixed